REFERENCE.md

```diff
@@ -189,3 +189,11 @@ Default value: [
     '--disablesssdauth',
   ]
 
+##### `authselect_profile`
```

Comment:
Don't modify this file. it is auto generated during the deployment process.

Reply:
dropped